client/src/components/App.jsx

```diff
@@ -20,7 +20,6 @@ class App extends React.Component {
   }
 
   handleReviews() {
-    console.log(this.state.randRestID)
     axios.get(`/${this.state.randRestID}/reviews`)
```

Comment:
Lint isn't bugging you to destructure this?

Reply:
hmm...that's weird that it isn't..